src/Google.Maps/GoogleSigned.cs

```diff
@@ -48,6 +50,9 @@ public static GoogleSigned SigningInstance
 
 		public string ClientId { get { return _clientId; } }
 
+		public string ReferralUrl { get { return _referralUrl; } }

```

Comment:
Also, I didnt see any mention of Referral Url in the signing documents: 
https://developers.google.com/maps/documentation/maps-static/get-api-key

Reply:
Is this for apikey filtering?  It would be nice to call that out, and add that as documentation for the additional property.